src/main/java/pl/edu/agh/firecell/model/material/ConductionCoefficientProvider.java

```diff
@@ -0,0 +1,38 @@
+package pl.edu.agh.firecell.model.material;
+
+import java.util.HashMap;
+import java.util.Optional;
+
+public class ConductionCoefficientProvider {
```

Comment:
Provider is a name for a different thing. Use name `MaterialConductionMap`

Reply:
what thing?